__tests__/app.test.js

```diff
@@ -75,22 +75,39 @@ describe("/api/articles", () => {
       });
 });
 });  
-
-
-
-
-
-
-
-
-
-
-
-// test("200: Responds with a new property of comment_count which has the correct number of comments by aricle_id", () => {
-//         return request(app)
-//         .get("/api/articles/1")
-//         .expect(200)
-//         .then(({body}) => {
-//           expect(body.article.comment_count).toBe("11")
-//         })
-//       })
+describe("/api/articles/:article_id", () => {
+  describe("GET", () => {
+    test("200: Responds with an article object with its properties", () => {
+      return request(app)
+      .get(`/api/articles/1`)
+      .expect(200)
+      .then(({body : {article}}) => {
+        expect(article).toEqual({
```

Comment:
Consider using a more flexible jest matcher here such as toMatchObject or objectContaining. If our database is ever updated we don't want to go back and edit our tests they should always pass.